tests/test_network_consistency.py

```diff
@@ -61,7 +61,13 @@ def test_network_consistency(self, net_name, data_path, json_path):
 
         # Create model
         model = nets.__dict__[net_name](**model_params)
-        model.load_state_dict(loaded_data["model"])
+        state_dict = loaded_data["model"]
+        model_dict = model.state_dict()
+        state_dict = {
+            k: v for k, v in state_dict.items() if (k in model_dict) and (model_dict[k].shape == state_dict[k].shape)
+        }
+        model_dict.update(state_dict)
+        model.load_state_dict(model_dict)
```

Comment:
Can you load weights directly from `state_dict` without the filtering if you set the `strict` argument to `False`?

Reply:
Thanks, this method works